src/main/resources/io/rpg/viewmodel/location-view.fxml

```diff
@@ -7,7 +7,7 @@
 
 <HBox fx:id="parent" alignment="CENTER" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="640.0" prefWidth="640.0" style="-fx-background-color: #cb2f2f;" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1" fx:controller="io.rpg.viewmodel.LocationViewModel">
    <children>
-      <Pane fx:id="contentPane" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="640.0" prefWidth="640.0">
+      <Pane fx:id="contentPane" viewOrder="1" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="640.0" prefWidth="640.0">
```

Comment:
Z czystej ciekawośCi: co robi ten `viewOrder="1"`?

Reply:
to jest taki jakby z-index, tzn kolejnosc, w ktorej sa wyswietlane rzeczy (np jesli sa dwa obiekty w tym samym miejscu to ten z wiekszym viewOrder zasloni ten z mniejszym)
teraz w sumie juz niepotrzebny, wyrzuce, to zostalo po tym, jak zrobilem fxml dla kazego gameobjectu i chcialem zeby byly rysowane na tle, a nie na odwrot